src/main/java/seedu/address/logic/commands/FindTagCommand.java

```diff
@@ -0,0 +1,42 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.person.TagContainsKeywordsPredicate;
+
+/**
+ * Finds and lists all contacts in address book whose tag contains any of the argument keywords.
+ * Keyword matching is case insensitive.
+ */
+public class FindTagCommand extends Command {
+
+    public static final String COMMAND_WORD = "findtag";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all contacts whose tag contain any of "
```

Comment:
Slight grammer error: "whose tags contain"

Reply:
Thanks for picking this out 😌